apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2Test.java

```diff
@@ -271,6 +274,49 @@ public void mapEobCommonGroupInpOutHHAHospiceSNFWhenNoFiNumberExpectNoFiNumExten
     assertNull(fiNumExtension);
   }
 
+  /** Ensures the organization contained resource is mapped correctly in the eob. */
+  @Test
+  public void mapEobCommonGroupInpOutHHAHospiceSNFWhenEOBHasAContainedOrganization() {
+
+    ExplanationOfBenefit eob = new ExplanationOfBenefit();
+    Instant instant = Instant.now();
+
+    TransformerUtilsV2.mapEobCommonGroupInpOutHHAHospiceSNF(
+        eob,
+        Optional.of(NPIOrgLookup.FAKE_NPI_NUMBER),
+        Optional.of(NPIOrgLookup.FAKE_NPI_ORG_NAME),
+        ' ',
+        ' ',
+        Optional.empty(),
+        "",
+        ' ',
+        Optional.empty(),
+        BigDecimal.ZERO,
+        BigDecimal.ZERO,
+        Optional.empty(),
+        Optional.of(instant),
+        Optional.empty(),
+        Optional.empty(),
+        C4BBInstutionalClaimSubtypes.Inpatient,
+        Optional.empty());
+
+    assertNotNull(eob.getContained());
+    Organization actualEobContainedOrganizationResource = (Organization) eob.getContained().get(0);
```

Comment:
Might be worth a getContained size assertion to make sure its 1 (duplicate mapping check), also avoids an exception if it the container to contained exists but has no elements.